draft-ietf-quic-transport.md

```diff
@@ -1743,6 +1744,10 @@ it can associate the peer's response with the corresponding PATH_CHALLENGE.
 On receiving a PATH_CHALLENGE frame, an endpoint MUST respond immediately by
 echoing the data contained in the PATH_CHALLENGE frame in a PATH_RESPONSE frame.
 
+An endpoint MUST NOT send more than one PATH_RESPONSE frame in response to one
+PATH_CHALLENGE frame (see {{retransmission-of-information}}).  A new
+PATH_CHALLENGE frame will be sent if another PATH_RESPONSE frame is needed.
```

Comment:
Suggested rephrase: The peer is expected to send more PATH_CHALLENGE frames as necessary to evoke additional PATH_RESPONSE frames.

Reply:
Nice! Updated in both places for consistency (3c0d1f7).